docs/b2b-edition/specs/storefront/storefront/order.yaml

```diff
@@ -1500,7 +1529,7 @@ paths:
                           - shipments
                           - poNumber
                           - money
-                      - $ref: https://raw.githubusercontent.com/bigcommerce/docs/main/docs/b2b-edition/models/order/order-extra.yaml
+                      - $ref: '../../../models/order/order-extra.yaml'
```

Comment:
Why was this update? I think it's clearer to have the full URL here. 🤔

Reply:
The full URL causes errors. The errors appear in visual studio. I had to update all of them.